static/app/components/organizations/projectPageFilter/index.tsx

```diff
@@ -444,9 +444,9 @@ export function ProjectPageFilter({
 }
 
 function shouldCloseOnInteractOutside(target: Element) {
-  // Don't close select menu when clicking on power hovercard ("Requires Business Plan")
-  const powerHovercard = document.querySelector("[data-test-id='power-hovercard']");
-  return !powerHovercard?.contains(target);
+  // Don't close select menu when clicking on power hovercard ("Requires Business Plan") and disabled feature hovercard that includes a link to the configuration documentation
+  const powerHovercard = target.closest('.power-hovercard');
+  return !powerHovercard;
```

Comment:
this is sus, i see that `data-test-id="power-hovercard"` still exists. `shouldCloseOnInteractOutside` doesn't have much to do with z-index, seems like a separate task & PR

Reply:
this change was to fix the action of pressing the link in the disabled tooltip, fair, will move to a separate PR, thanks for checking y'all!